discord_bots/commands.py

```diff
@@ -3033,20 +3022,30 @@ async def setgamecode(ctx: Context, code: str):
         )
         return
 
-    if ipg.code:
-        await send_message(
-            ctx.message.channel,
-            embed_description=f"Game **{short_uuid(ipg.id)}** already has a code! Code: **{ipg.code}**",
-            colour=Colour.red(),
-        )
-        return
-
     ipg.code = code
+    ipg_players: list[InProgressGamePlayer] = (
+        ctx.session.query(InProgressGamePlayer)
+        .filter(
+            InProgressGamePlayer.in_progress_game_id == ipg.id,
+            # InProgressGamePlayer.player_id != ctx.message.author.id
```

Comment:
```suggestion
```

Reply:
Oops, forgot to include this line. It excludes the message author from being send a DM of the gamecode since they already know it and made the lobby